### PR TITLE
[FIX] Satisfies lint

### PR DESCRIPTION
**Pull request by filipedelimabrito**
It is unable to build the project since we are using the Kotlinter and the task was failing (due to the non-following code style).

./gradlew build.

### CODE REVIEW

**Comment by antgel** [comment]
Sorry for the noise. It was failing in Android Studio. When I ran `./gradlew clean && `./gradlew assemble`, it worked. I'd love to know why if anyone has a clue.

**Comment by filipedelimabrito** [comment]
> Build still fails for me.

What's was going wrong prior this PR? What error you were getting?

**Comment by antgel** [comment]
@filipedelimabrito It works when I run the commands from the terminal, which I have no problem doing. :)

File: core/src/test/kotlin/chat/rocket/core/internal/rest/UserTest.kt [hunk 1]
```diff
@@ -91,18 +91,18 @@ class UserTest {
...
+//        mockServer.expect()
+//                .get().withPath("/api/v1/rooms.get?updatedAt=1970-01-01T00:00:00.000Z").andReturn(200, ROOMS_OK).once()
+//        mockServer.expect()
+//                .get().withPath("/api/v1/subscriptions.get?updatedAt=1970-01-01T00:00:00.000Z").andReturn(200, SUBSCRIPTIONS_OK).once()
+//
+//        runBlocking {
+//            val rooms = sut.chatRooms()
+//            System.out.println("Rooms: $rooms")
+//        }
+//    }
```
Comment (leonardoaramaki) [inline core/src/test/kotlin/chat/rocket/core/internal/rest/UserTest.kt]: Why is this test commented?

Comment (luciofm) [reply core/src/test/kotlin/chat/rocket/core/internal/rest/UserTest.kt]: It is failing... No idea why... Will investigate

**Pull request merged**